posthog/permissions.py

```diff
@@ -257,6 +259,31 @@ def has_permission(self, request, view) -> bool:
         return False
 
 
+class TimeSensitiveActionPermission(BasePermission):
```

Comment:
So clear - nice work!

Reply:
🙌 Fixed the issue in your comment. (reminder that commenting issues as a line comment verus a review comment are easier to reply to inline 👍 )